roles/install_nextcloud/README.md

```diff
@@ -112,19 +112,24 @@ nextcloud_archive_format: "zip" # zip | tar.bz2
 Choose between the 2 archive formats available in the repository.
 
 ```yaml
-# nextcloud_full_url:
+# nextcloud_full_src:
 ```
 
 _If you don't like rules..._
 Specify directly a path or full URL to the archive. The role will skip the url generation and copy from the control-host or download the archive. **Requires nextcloud_version_major to be set along**.
 
 #### Examples:
-- Download your own archive:
+- Download or Copy your own archive:
   (_you **must** specify the nextcloud major version along_)
 
 ```yaml
-nextcloud_full_url: https://download.nextcloud.com/server/releases/nextcloud-25.0.0.zip
-nextcloud_version_major: 42
+# Download via URL
+nextcloud_full_src: https://download.nextcloud.com/server/releases/nextcloud-25.0.0.zip
+nextcloud_version_major: 25
+# Copy from control host
+nextcloud_full_src: "files/nextcloud-27.0.0.zip"
+nexcloud_version_major: 27
```

Comment:
No need to duplicate the code.
`nexcloud_version_major` can be moved above these 2 variables.

Reply:
This is correct. My intention was to indicate to the reader, that `nextcloud_version_major` must always be set, even though ansible could derive the major version from the filename. Your comment evidences, that this did not work out. Moving up would not work, because on the one hand side, there is version 25 and on the other 27. Have you got any suggestion how to improve the description?

Reply:
Got it! It makes sense in this case if it's just to emphasize. And as it's only the README we can treat it as example.